apps/indexer-coordinator/src/project/project.service.ts

```diff
@@ -220,6 +221,8 @@ export class ProjectService {
       postgres,
       mmrStoreType,
       dockerNetwork,
+      ipfsUrl: IPFS_URL,
+      mmrPath: argv['mmrPath'],
```

Comment:
[suggestion] better format the `mmrPath` to remove the end `/`, just avoid user input path such as `/home/` which will make the node failed

Reply:
Code updated to make sure no trailing `/` in the `mmrPath`.